storage/src/test/java/org/apache/kafka/server/log/remote/storage/LocalTieredStorageEvent.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.server.log.remote.storage;
+
+import org.apache.kafka.common.TopicPartition;
+
+import java.util.Optional;
+
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+import static java.util.Optional.ofNullable;
+
+/**
+ * Represents an interaction between a broker and a second-tier storage. This type of event is generated
+ * by the {@link LocalTieredStorage} which is an implementation of the {@link RemoteStorageManager}
+ * operating in Kafka's runtime as the interface between Kafka and external storage systems, through
+ * which all such interactions go through.
```

Comment:
Sorry, the last sentence is hard to understand: `through which all such interactions go through.` 
Is there better way to describe it?